blueprints/ember-spread/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-prop-types', target: '^3.0.0'}
+        {name: 'ember-prop-types', target: '^3.14.1'}
```

Comment:
Since ember-prop-types and its dependencies have no blueprints to run, can we move this to dependency?